### PR TITLE
LPS-103354 | Not available estimation

### PR DESCRIPTION
**Pull request by andresfulla**

### CODE REVIEW

**Comment by andresfulla** [comment]
ci:test:sf

**Comment by andresfulla** [comment]
ci:test:relevant

File: modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js [hunk 1]
```diff
@@ -187,29 +186,38 @@ function ReviewExperimentModal({onRun, setVisible, variants, visible}) {
...
+											'time-depends-on-confidence-level-and-traffic-to-the-variants'
 										)}
 									</p>
 								</div>
-								<p className="small text-secondary">
-									{Liferay.Language.get(
-										'time-depends-on-confidence-level-and-traffic-to-the-variants'
+
+								<p className="mb-0 text-nowrap">
+									{estimation.loading ? (
```
Comment (dgarciasarai) [inline modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js]: Crazy ternary

File: modules/dxp/apps/segments/segments-experiment-web/test/js/components/ReviewExperimentModal.es.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+		beforeAll(() => {
+			jest.useFakeTimers();
+		});
+
+		afterEach(() => {
+			cleanup();
+		});
+
+		it('Triggers on first render', async () => {
```
Comment (dgarciasarai) [inline modules/dxp/apps/segments/segments-experiment-web/test/js/components/ReviewExperimentModal.es.js]: test()

**Comment by andresfulla** [comment]
to be rebased...

**Comment by andresfulla** [comment]
ci:test:sf

**Comment by andresfulla** [comment]
ci:test:relevant

**Comment by andresfulla** [comment]
ci:test:sf

**Comment by andresfulla** [comment]
ci:test:relevant

**Pull request closed without merging**